learning/katas/go/Core Transforms/Map/ParDo/task.md

```diff
@@ -0,0 +1,39 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# ParDo
+
+ParDo is a Beam transform for generic parallel processing. The ParDo processing paradigm is similar to the “Map” 
+phase of a Map/Shuffle/Reduce-style algorithm: a ParDo transform considers each element in the input PCollection,
+performs some processing function (your user code) on that element, and emits zero, one, or multiple elements to an 
+output PCollection.
+
+**Kata:** Please write a simple ParDo that maps the input element by multiplying it by 10.
```

Comment:
A possible adjacent task/step is converting a func DoFn into a Structural DoFn. Yes, this is relatively simple, but from a learning standpoint, it makes the distinction pretty clear, while not asking users too much else that might conflate with it.  (eg. Getting the wrong idea that funcs must be 1:1 vs 1:many/none, vs structs etc).

Not necessary to do it in this PR.

Reply:
@lostluck, the space will not create any problem. It is friendlier for the learner to see the lesson and task names in the natural way versus using underscore name.